src/index.html

```diff
@@ -2,16 +2,152 @@
+          How old are You?
+          <input
+            class="form-number"
+            type="number"
+            id="number"
+            value="12"
+            min="0"
+            max="100"
+          >
+        </label>
+        <label class="label"  for="formdate">
+          Full date of birth:
+          <input
+            id="formdate"
+            type="date"
+            name="formdate"
+          >
+        </label>
+        <label class="label" for="accept">
+          I accept the term of the agreement
+          <input id="accept" type="checkbox">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="label" for="email">
+          E-mail:
+          <input
+           name="email"
+           id="email"
+           type="email"
+           placeholder="email@example.com"
+          >
+        </label>
+        <label class="label" for="password">
+          Password:
+          <input id="password" type="password">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <label class="label" for="cats">
+          Do you love cats?
+          <label for="cats-yes">
+            <input
+              id="cats-yes"
+              type="radio"
+              name="cats"
+            >
+            Yes
+          </label>
+          <label for="cats-no">
+            <input
+            id="cats-no"
+            type="radio"
+            name="cats"
+            >
+            No
+          </label>
+        </label>
+        <label class="label" for="color">
+         What is your favorite color?
+          <input
+            name="color"
+            id="color"
+            type="color"
+          >
+        </label>
+        <label class="label" for="datelocal">
+          What time do you go to bed?
+          <input
+            id="datelocal"
+            type="datetime-local"
+          >
+        </label>
+        <label class="label" for="Cars">
+          What are your favorite brand of cars?
+          <select
+            name="Car"
+            required multiple
+            id="Cars"
+          >
+            <option value="1">BMW</option>
+            <option value="2">Audi</option>
+            <option value="3">Lada</option>
+          </select>
+        </label>
+        <label class="label" for="form-work">
+          How do you rate our work?
+          <input
+            id="form-work"
+            type="range"
+            min="1"
+            max="100"
+            value="1"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Additional info:</legend>
+        <label class="label" for="textarea">
+        Comments:
+          <textarea
+            id="textarea"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
+        </label>
```

Comment:
```suggestion
          <textarea
            id="textarea"
            cols="20"
            rows="2"
          ></textarea>
        </label>
```

Still not fixed 
![image](https://user-images.githubusercontent.com/70009205/215256066-e61b6502-3462-4299-806a-e502847dcc45.png)

Reply:
fix it. welcome to write in the chat if you have questions about comments
![image](https://user-images.githubusercontent.com/98764641/225850327-bf823730-9684-4bfe-ac56-654ba848f400.png)